src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/prisonperson/PrisonPersonIntTest.kt

```diff
@@ -39,27 +40,25 @@ class PrisonPersonIntTest(
         prisonPersonDpsApi.stubGetPrisonPerson(prisonerNumber = "A1234AA")
         prisonPersonNomisApi.stubPutPhysicalAttributes(offenderNo = "A1234AA")
 
-        service.updatePhysicalAttributes(offenderNo = "A1234AA")
+        awsSnsClient.publish(physicalAttributesRequest()).get()
+          .also { waitForAnyProcessingToComplete() }
       }
 
       @Test
-      fun `should call DPS API`() {
+      fun `should update physical attributes`() {
+        // should call DPS API
         prisonPersonDpsApi.verify(
           getRequestedFor(urlPathEqualTo("/prisoners/A1234AA")),
         )
-      }
 
-      @Test
-      fun `should call NOMIS API`() {
+        // should call NOMIS API`
```

Comment:
Decided that I don't like a separate test for each assertion, takes 3 times as long.